geonode/static/geonode/css/base.css

```diff
@@ -8176,3 +8176,45 @@ form-inline {
   border: 1px solid #ccc;
   border-radius: 4px;
 }
+.geonode-slide-show .jumbotron {
+  padding: 6em 0px 2em 0px !important;
```

Comment:
the use of !important  should be avoided where possible.
- in case it is not needed please remove it
- in case it is needed consider to increase the specificity of your selector in best case as BEM naming